package proto_ff;

import "yd_fieldoptions.proto";


message pathpath
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "路径ID"];
	optional int64 belongToSceneID = 2[(yd_fieldoptions.field_cname) = "所属场景"];
}


message Sheet_pathpath
{
	repeated pathpath pathpath_List = 1[(yd_fieldoptions.field_arysize)=20];
}
